runtime/src/accounts.rs

```diff
@@ -619,8 +619,13 @@ impl Accounts {
             .collect()
     }
 
-    pub fn calculate_capitalization(&self, ancestors: &Ancestors) -> u64 {
-        self.accounts_db.unchecked_scan_accounts(
+    pub fn calculate_capitalization(&self, ancestors: &Ancestors, slot: Slot) -> u64 {
+        let cap = self
+            .accounts_db
+            .update_accounts_hash_with_index_option(false, false, slot, ancestors, None, true)
+            .1;
```

Comment:
we could also pass a parameter here to prevent actually calculating a hash value since we throw it away.

Reply:
Also, I need to change this to call a level deeper and not update the hash - but just calculate the hash to maintain the behavior of the function I'm attempting to replace.